daemon/libnetwork/ipams/drivers.go

```diff
@@ -22,10 +22,8 @@ func Register(r ipamapi.Registerer, pg plugingetter.PluginGetter, lAddrPools, gA
 	if err := null.Register(r); err != nil {
 		return err
 	}
-	if pg != nil {
-		if err := remoteIpam.Register(r, pg); err != nil {
-			return err
-		}
+	if err := remoteIpam.Register(r, pg); err != nil {
```

Comment:
If I get it correctly, pg is nil on Windows because it doesn't support 'managed' plugins, right? Would you mind adding a comment telling that, so we don't reintroduce this issue?